doc/rtd/topics/datasources/aliyun.rst

```diff
@@ -12,6 +12,22 @@ The Alibaba Cloud metadata service is available at the well known url
 Alibaba Cloud ECS on `metadata
 <https://www.alibabacloud.com/help/zh/faq-detail/49122.htm>`__.
 
+Configuration
+-------------
+The following configuration can be set for the datasource in system
+configuration (in `/etc/cloud/cloud.cfg` or `/etc/cloud/cloud.cfg.d/`).
+
+An example configuration with the default values is provided below:
+
+.. sourcecode:: yaml
+
+  datasource:
+    AliYun:
+      metadata_urls: ["http://100.100.100.200", "http://instance-data:8773"]
+      support_xen: false
+      timeout: 5
+      max_wait: 300
```

Comment:
These don't match the AliYun defaults (which are inherited from the EC2 datasource): https://github.com/canonical/cloud-init/blob/e5c67478136e1942a981ee18785ceb26579d0041/cloudinit/sources/DataSourceEc2.py#L69-L70